src/os/ui/query/queryhandler.js

```diff
@@ -298,20 +310,22 @@ os.ui.query.QueryHandler.prototype.createFilter = function() {
     entries.push(entry);
   }
 
-  // ignore disabled areas
-  entries = entries.filter(os.ui.query.QueryHandler.shownAreas);
+  if (!opt_ignoreShown) { // ignore disabled areas
+    entries = entries.filter(os.ui.query.QueryHandler.shownAreas.bind(null, am));
```

Comment:
I'm seeing why you had the manager parameter first. The bind can easily be removed while still having a concise one-liner by using an arrow function:

```
entries = entries.filter((entry) => os.ui.query.QueryHandler.shownAreas(entry, am));
```